openapi.go

```diff
@@ -164,7 +193,7 @@ func validateSwaggerUrl(swaggerUrl string) bool {
 }
 
 // RegisterOpenAPIOperation registers an OpenAPI operation.
-func RegisterOpenAPIOperation[T, B any](s *Server, route Route[T, B]) (*openapi3.Operation, error) {
+func RegisterOpenAPIOperation[T, B any](s *OpenAPI, route Route[T, B]) (*openapi3.Operation, error) {
 	if route.Operation == nil {
```

Comment:
Enables the possibility to do `RegisterOpenAPIOperation[T, B](fuegoGinCompat.OpenAPI, route)`